pkgs/by-name/ab/aba/package.nix

```diff
@@ -0,0 +1,53 @@
+{ fetchFromSourcehut
+, just
+, lib
+, nix-update-script
+, rustPlatform
+, scdoc
+}:
+let
+  version = "0.7.0";
+in
+rustPlatform.buildRustPackage {
+  pname = "aba";
+  inherit version;
+
+  src = fetchFromSourcehut {
+    owner = "~onemoresuza";
+    repo = "aba";
+    rev = version;
+    hash = "sha256-YPE5HYa90BcNy5jdYbzkT81KavJcbSeGrsWRILnIiEE=";
+    domain = "sr.ht";
+  };
+
+  cargoSha256 = "sha256-wzI+UMcVeFQNFlWDkyxk8tjpU7beNRKoPYbid8b15/Q=";
+
+  nativeBuildInputs = [
+    just
+    scdoc
+  ];
+
+  # There are no tests
+  doCheck = false;
```

Comment:
Unless the build support function crashes as a result, I believe the default way to handle this is to just leave it on.

Reply:
I think that Rust packages are an exception to such rule. The [nixpkgs manual](https://nixos.org/manual/nixpkgs/stable/#rust) on the `Disabling package-tests` section states:

>  In some instances, it may be necessary to disable testing altogether (with doCheck = false;):
> 
> - If no tests exist – the checkPhase should be explicitly disabled to skip unnecessary build steps to speed up the build.